templates/docker/Dockerfile

```
@@ -43,8 +43,8 @@ RUN apt-get update \
 
 ## NOTE(neoaggelos): Go version used to build components
 ## !!!IMPORTANT!!! Keep up to date with "snapcraft.yaml:parts.build-deps.build-snaps.go"
-ADD install-go.sh
-RUN install-go.sh 1.22 && ln -s /usr/local/go/bin/go /usr/local/bin/go
+ADD install-go.sh /
+RUN ./install-go.sh 1.22 && ln -s /usr/local/go/bin/go /usr/local/bin/go
```

```suggestion
RUN /install-go.sh 1.22 && ln -s /usr/local/go/bin/go /usr/local/bin/go
```
Wouldn't `./` break if `WORKDIR` changes?

---

true